azure-functions/create-function-app-connect-to-storage/create-function-app-connect-to-storage-account.sh

```diff
@@ -9,6 +9,7 @@ let "randomIdentifier=$RANDOM*$RANDOM"
 location="eastus"
 resourceGroup="msdocs-azure-functions-rg-$randomIdentifier"
 tag="create-function-app-connect-to-storage-account"
+storage="msdocsaccount$randomIdentifier"
```

Comment:
@alexandair, you need a dash after "msdocs" like this "msdocs-account".  However, where is this parameter used in the script?

Reply:
@dbradish-microsoft pretty sure that non-alpha characters aren't allowed in Storage account names. `$storage` is used in at least 2 commands.

Reply:
Storage account names must only use numbers and lowercase letters. No uppercase letters or special characters.
That's why I've picked `msdocsaccount$randomIdentifier` as a name.